src/qt/test/wallettests.cpp

```diff
@@ -144,7 +144,12 @@ void TestGUI()
         LOCK(cs_main);
         WalletRescanReserver reserver(wallet.get());
         reserver.reserve();
-        wallet->ScanForWalletTransactions(chainActive.Genesis(), nullptr, reserver, true);
+        const CBlockIndex* const null_block = nullptr;
+        const CBlockIndex* stop_block = nullptr;
```

Comment:
nit: No need to initialize a return value. In fact this might look like you are passing in something.

Also could inline nullptr for null_block below for brevity.

Reply:
Removed the initialization of the out var, though this required setting the out var to null in the method for consistent behavior under test. Seems like a good change to me as this seems more predictable.

Can't inline the `null_block` vars as that results in template ambiguity around `<ostream, nullptr_t>`